test/testobjects/testconstants.ts

```diff
@@ -0,0 +1,25 @@
+export class TestConstants {
+    public static readonly connectionPlaceholder: string = "CONNECTION";
+    public static readonly taskHubPlaceholder: string = "TASK-HUB";
+    public static readonly hostPlaceholder: string = "HOST-PLACEHOLDER";
+
+    public static readonly testCode: string = "code=6PDWS6KrNPAOv/hw7cTDgiabhQwKTQQEnEZptmIUawunGPpOFvzDRQ==";
```

Comment:
Does this `code` have to look like a real secret?  I worry about GitHub code scanners flagging this.

Reply:
Ah, good point. It doesn't; will replace.